db/migrate/20210203093153_add_correct_to_short_problems.rb

```diff
@@ -0,0 +1,5 @@
+class AddCorrectToShortProblems < ActiveRecord::Migration
+  def change
+    add_column :short_problems, :correct, :integer, default: 1
```

Comment:
Can you change these to `correct_score` `wrong_score` `empty_score`? I think these longer names make them much clearer.

Reply:
okay